pcs_detection/scripts/data/aluminum_3/training_labels.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we rename this dataset to example_dataset_1 or something similar. That's why I had those dataset directories. Also, could we split them into 2 separate datasets to show how to combine multiple?

Reply:
It doesn't have to be anything special. You could just copy the second half into the other directory